js/app.js

```diff
@@ -58,6 +59,21 @@ var App = function() {
 		}
 	};
 
+	var checkDeprecatedOptions = function(userConfig) {
+		var deprecatedOptions = require(__dirname + "/../config/deprecated.js");
```

Comment:
Why not used into js directory?. I think the config dir is more touched by the users and they can be replace the content inside it. If remove the `deprecated.js` file can be raise a error. For me have more sense, using _deprecated_ into the core directory, may be in the future we can be put functions or other stuff to deprecated not just configuration directives.